custom_model_runner/datarobot_drum/runtime_parameters/runtime_parameters.py

```diff
@@ -0,0 +1,42 @@
+"""
+Copyright 2023 DataRobot, Inc. and its affiliates.
+All rights reserved.
+This is proprietary source code of DataRobot, Inc. and its affiliates.
+Released under the terms of DataRobot Tool and Utility Agreement.
+"""
+import json
+import os
+import trafaret as t
+
+from datarobot_drum.runtime_parameters.runtime_parameters_schema import (
+    RuntimeParameterPayloadTrafaret,
+)
+from datarobot_drum.runtime_parameters.exceptions import InvalidJsonException
+from datarobot_drum.runtime_parameters.exceptions import InvalidRuntimeParam
+
+
+class RuntimeParameters:
```

Comment:
Looks like it becomes the first user facing DRUM public API. Docs should be added here.